modules/olm-installing-operators-from-operatorhub.adoc

```diff
@@ -43,11 +33,6 @@ ifndef::olm-user[]
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 Installation Mode:: Choose *All namespaces on the cluster (default)* to have the Operator installed on all namespaces or choose individual namespaces, if available, to only install the Operator on selected namespaces. This example chooses *All namespaces...* to make the Operator available to all users and projects.
 endif::[]
-ifdef::openshift-dedicated[]
-Installation Mode:: In {product-title} clusters, you can choose *All namespaces on the cluster (default)* to have the Operator installed on all namespaces. This makes the Operator available to all users and projects.
-endif::[]
-endif::[]
```

Comment:
@EricPonvelle can you please review this edit? It might be that this line and the next one should not be removed. The [4.9](https://github.com/openshift/openshift-docs/pull/39080/files#diff-a715e94f710e17de07d7b0142921d9d7f6b2bbfb3b25f9e5cfcf1e8a03023d1dR36-R37) and [4.10](https://github.com/openshift/openshift-docs/pull/39069/files#diff-a715e94f710e17de07d7b0142921d9d7f6b2bbfb3b25f9e5cfcf1e8a03023d1dR36-R37) PRs keep these lines in place. Thanks!

Reply:
@pneedle-rh I think I fixed it. Great catch! This took me a bit to unravel. Can you take a look and make sure I didn't break it further? Thanks!

Reply:
That looks good to me, thanks!